config/initializers/message_bus.rb

```diff
@@ -1,19 +1,34 @@
 # frozen_string_literal: true
 
-if Rails.env.test?
-  MessageBus.configure(backend: :memory)
-  MessageBus.keepalive_interval = 0
-  MessageBus.off
-else
-  MessageBus.redis_config = System::Application.config.redis.merge(db: 8)
-  MessageBus.timer.on_error do |error|
-    System::ErrorReporting.report_error(error)
+class MessageBusConfig
```

Comment:
maybe create an anonymous class as this is in the initializer?

Reply:
I did not see that :)